experimental/dds/tree2/api-report/tree2.api.md

```diff
@@ -1983,24 +1986,32 @@ export class SharedTreeFactory implements IChannelFactory {
 }
 
 // @alpha
-export interface SharedTreeList<TTypes extends AllowedTypes, API extends "javaScript" | "sharedTree" = "sharedTree"> extends ReadonlyArray<ProxyNodeUnion<TTypes, API>> {
+export type SharedTreeList<TTypes extends AllowedTypes, API extends "javaScript" | "sharedTree" = "sharedTree"> = readonly ProxyNodeUnion<TTypes, API>[] & SharedTreeNode & {
     insertAt(index: number, value: Iterable<FlexibleNodeContent<TTypes>>): void;
```

Comment:
The 'new' bit is the "& SharedTreeNode &"...  the rest is api-extract resorting the members.